esp/esp/program/models/__init__.py

```diff
@@ -281,6 +281,7 @@ class Program(models.Model, CustomFormsLinkModel):
     class Meta:
         app_label = 'program'
         db_table = 'program_program'
+        ordering = ['-id','name']
```

Comment:
Not sure what the point of the `name` here is, since `id`s should be unique.  It might make some queries slower, I'm not sure.